internal/store/builder.go

```diff
@@ -95,6 +96,18 @@ func (b *Builder) WithEnabledResources(r []string) error {
 		if !resourceExists(col) {
 			return fmt.Errorf("resource %s does not exist. Available resources: %s", col, strings.Join(availableResources(), ","))
 		}
+
+		deprecationNoticeHeader := "DEPRECATION NOTICE"
```

Comment:
Instead of adding the deprecation notice here, we should reuse the existing mechanism in the family generators: https://github.com/kubernetes/kube-state-metrics/blob/master/pkg/metric_generator/generator.go#L30-L37